lib/src/app.dart

```diff
@@ -12,9 +13,9 @@ class _AppState extends State<App> {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      theme: ThemeData(primarySwatch: Colors.pink),
-      darkTheme: ThemeData.dark(),
-      home: const MainPage(title: 'Quickgui - A Flutter frontend for Quickget and Quickemu'),
-    );
+        theme: yaru.lightTheme,
```

Comment:
I think we wanted to retain the pink accent colours, though I might be mistaken?

Reply:
Nop, you are absolutely right ! The pink is here to stay ! :-)